src/pages/ProfilePage.jsx

```diff
@@ -2,9 +2,10 @@ import React from "react";
 
 class ProfilePage extends React.Component {
   renderAbout(name) {
-    const obj = {
+    var obj = {
       'gabs': <p>Gabs, O hater de switchs e ator nas horas vagas</p>,
       'icaro': <p>Icaro, o especiclista.</p>,
+      'rapunzel': <h2>Rapunzel, o Bruxo dos códigos</h2>
```

Comment:
O que você acha de ficar no padrão?
```suggestion
      'rapunzel': <p>Rapunzel, o Bruxo dos códigos</p>
```